src/network/network_type.h

```diff
@@ -74,18 +74,23 @@ enum NetworkPasswordType {
 	NETWORK_COMPANY_PASSWORD, ///< The password of the company.
 };
 
-/** Destination of our chat messages. */
+/**
+ * Destination of our chat messages.
+ * @warning The values of the enum items are part of the admin network API. Only append at the end.
```

Comment:
technically, "Only append at the end" is redundant. "Only append, do not insert into the enum" might be better